realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -151,44 +151,6 @@ public void basic_multipleReferencesFromParentList() {
         assertEquals(parent, child.getListParents().last());
     }
 
-    // A listener registered on the backlinked object should be called when a commit adds a backlink
-    @Test
-    @RunTestInLooperThread
-    public void notification_onCommitModelObject() {
```

Comment:
no, it is not. And it is expected behavior from object store.
Line 175 called `parent.setFieldObject(child);`, the changed object is parent, but the original test added listener on `child` which won't be trigger since nothing changed on the `child`. I am not quite sure what is the purpose of this test, maybe @bmeike can help to clarify this.